services/identity/__tests__/MailClient.spec.ts

```diff
@@ -39,10 +53,15 @@ describe("Mail Client", () => {
     // Store the API key and set it later so that other tests are not affected
     const curApiKey = process.env.POSTMAN_API_KEY
     process.env.POSTMAN_API_KEY = ""
+    // NOTE: This is because of typescript transpiling down to raw js
+    // Export default compiles down to module.exports.default, which is also
+    // done by babel.
+    // Read more here: https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports
+    const _MailClientWithoutKey = (await import("../MailClient")).default
```

Comment:
This is ok for now - let's treat this as a work-in-progress. 

I believe the end state we should aim for is to write MailClient in a dependency injectable way, so that we can invoke it more easily in our tests. Could we write this as a TODO comment in the MailClient file?

Reply:
if by this you mean that the `POSTMAN_API_KEY` is given as a parameter to the constructor, i can do it here!

Reply:
It's ok, we can leave it for now and add that to the various third-party client constructors in a separate PR/issue